src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2458,6 +2459,19 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         public System.String TrimStart() { throw null; }
         public System.String TrimStart(char trimChar) { throw null; }
         public System.String TrimStart(params char[] trimChars) { throw null; }
+        public partial struct RuneEnumerator : System.Collections.Generic.IEnumerable<System.Text.Rune>, System.Collections.Generic.IEnumerator<System.Text.Rune>
```

Comment:
The next logical place would be `System.Text.StringRuneEnumerator`. Since very few people will ever use this type directly (most should access it via a _foreach_, which hides the type name) I'm fine with moving it out in order to keep the string class cleaner.